merge-incident-columns.ipynb

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
It also might be a good idea to `.gitignore` the `.ipynb` files as well. I think it's best that we leave the Jupyter notebook files out of the repository and simply leave the scripts.